AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -118,6 +120,15 @@ final class AWSInitialSyncOrchestrator: InitialSyncOrchestrator {
         completion?(.successfulVoid)
     }
 
+    private func dispatchSyncQueriesStarted(_ syncableModels: [Model.Type]) {
+        let modelTask = syncableModels.map { $0.modelName }
+
+        let syncQueriesStartedEvent = SyncQueriesStartedEvent(models: modelTask)
+        let syncQueriesStartedpayload = HubPayload(eventName: HubPayload.EventName.DataStore.syncQueriesStarted,
```

Comment:
`syncQueriesStartedEventPayload`

Reply:
Updated, also updated `outboxStatusPayload` to `outboxStatusEventPayload` in `outgoingMutationQueue.swift`